README.md

```diff
@@ -1,2 +1,7 @@
 # Juntos
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
+
+
+## <a href="https://github.com/pesto-students/juntos-ninja-2-beta/projects/1#card-61704804">Task</a>
```

Comment:
We don't need this header, please link the PR to the Github issue.

Reply:
this is done